FluentTc.Tests/Locators/BuildParameterTypeBuilderFormatTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentTc.Locators;
+using NUnit.Framework;
+
+namespace FluentTc.Tests.Locators
+{
+    [TestFixture]
+    public class BuildParameterTypeBuilderFormatTests
+    {
+        [TestCase(null, null, "checkbox")]
+        [TestCase(null, "kek", "checkbox uncheckedValue='kek'")]
+        [TestCase("lol", null, "checkbox checkedValue='lol'")]
+        [TestCase("lol", "kek", "checkbox checkedValue='lol' uncheckedValue='kek'")]
+        public void BuildParameterCheckboxTypeBuilder_FormatTest(string checkedValue, string uncheckedValue, string expected)
+        {
+            // Arrange
+            var testObject = new BuildParameterCheckboxTypeBuilder();
+            if (checkedValue != null)
+                testObject.WithCheckedValue(checkedValue);
+            if (uncheckedValue != null)
+                testObject.WithUncheckedValue(uncheckedValue);
+
+            // Act
+            var result = testObject.Build();
+
+            // Assert
+            Assert.That(result, Is.EqualTo(expected));
```

Comment:
It's common to use FluentAssertions for verifying test result in FluentTc project